pkgs/tools/networking/haproxy/default.nix

```diff
@@ -1,22 +1,39 @@
 { useLua ? true
 , usePcre ? true
 , withPrometheusExporter ? true
+, sslLibrary ? "wolfssl"
 , stdenv
 , lib
 , fetchurl
 , nixosTests
-, openssl
 , zlib
 , libxcrypt
-, lua5_3 ? null
-, pcre ? null
+, wolfssl-haproxy ? null
+, libressl ? null
+, quictls ? null
+, boringssl ? null
+, openssl ? null
+, lua5_4 ? null
+, pcre2 ? null
 , systemd ? null
```

Comment:
``? null`` is a legacy pattern with major pitfalls. It is not required and only hides bugs, please remove it.